packages/app/pages/_app.tsx

```diff
@@ -125,13 +126,23 @@ export default function MyApp({ Component, pageProps }: AppPropsWithLayout) {
     fetch('/api/config')
       .then(res => res.json())
       .then(_jsonData => {
+        // Set API url dynamically for users who aren't rebuilding
+        try {
+          const url = new URL(_jsonData.apiServerUrl);
+          if (url != null) {
+            apiConfigs.prefixUrl = url.toString().replace(/\/$/, '');
```

Comment:
oh I see. So the hack is we reset this `apiConfigs` global var

Reply:
well the real hack is that we force everything to read the config on every request as opposed to having it pre-baked in the ky server instance hah